core/db/2-schema.sql

```diff
@@ -138,11 +138,14 @@ ALTER TABLE ONLY messages
 
 CREATE TABLE inventory (
     call character varying(32) NOT NULL,
-    quantity integer DEFAULT 1 NOT NULL,
-    volume integer default null,
     title character varying(255),
+    checkout_period int default 1,
+    quantity integer DEFAULT 1 NOT NULL,
     is_reserve BOOLEAN DEFAULT FALSE, -- these are for items left by professors for us to check out
-    CONSTRAINT inventory_pkey PRIMARY KEY (call)
```

Comment:
We probably still want this--there should only ever be one instance of a call number (quantity allows us to specify copies)